lib/__tests__/standalone.test.js

```diff
@@ -107,6 +107,81 @@ it("standalone with non-existent-file and allowEmptyInput false should throw err
   })
 })
 
+it("standalone with two non-existent-files and allowEmptyInput false should throw error with code 80", () => {
+  const expectedError = new Error(`${fixturesPath}/non-existent-file.css and ${fixturesPath}/another-non-existent-file.css does not match any files`)
+  expectedError.code = 80
+
+  return standalone({
+    files: [ `${fixturesPath}/non-existent-file.css`, `${fixturesPath}/another-non-existent-file.css` ],
+    config: configBlockNoEmpty,
+    allowEmptyInput: false,
+  }).then(() => {
+    throw new Error("should not have succeeded")
+  }).catch(actualError => {
+    expect(actualError).toEqual(expectedError)
+  })
+})
+
+it("standalone with three or more non-existent-files and allowEmptyInput false should throw error with code 80", () => {
+  const expectedError = new Error(`${fixturesPath}/non-existent-file.css, ${fixturesPath}/another-non-existent-file.css and ${fixturesPath}/third-non-existent-file.css does not match any files`)
+  expectedError.code = 80
+
+  return standalone({
+    files: [ `${fixturesPath}/non-existent-file.css`, `${fixturesPath}/another-non-existent-file.css`, `${fixturesPath}/third-non-existent-file.css` ],
+    config: configBlockNoEmpty,
+    allowEmptyInput: false,
+  }).then(() => {
+    throw new Error("should not have succeeded")
+  }).catch(actualError => {
+    expect(actualError).toEqual(expectedError)
+  })
+})
+
+it("standalone with a glob that doesn't match anything and allowEmptyInput false should throw error with code 80", () => {
+  const expectedError = new Error(`${fixturesPath}/nodir/**/*.css does not match any files`)
+  expectedError.code = 80
+
+  return standalone({
+    files: [`${fixturesPath}/nodir/**/*.css`],
+    config: configBlockNoEmpty,
+    allowEmptyInput: false,
+  }).then(() => {
+    throw new Error("should not have succeeded")
+  }).catch(actualError => {
+    expect(actualError).toEqual(expectedError)
+  })
```

Comment:
With this Promise usage, the `should not have succeeded` error will actually get caught by the `catch` statement, I believe. Although the test will still fail, the source of failure will be less clear. Can we instead do 
```js
.then(() => { 
  throw new Error("should not have succeeded")
}, actualError => {
 ..
})
```

for all these tests? Does that make sense?